home-screen-widget/profiles/DEVELOPERS_PORTAL.md

```diff
@@ -0,0 +1,411 @@
+# HOME
+Home is a feature that surfaces relevant information by displaying a list of widgets in a new main tab on the mobile clients. 
+This gives people effortless access to tools and information that are contextually relevant to their work. 
+
+## Contents
+
+1. Architecture
+2. Home screen widget development
+3. Configuring the home screen for a tenant
+
+
+## 1. Architecture
+
+Home or the home screen (as we call it in the code) consists of a hybrid app which is displayed in a web view on the native clients (Android, iOS).
+From an architecture perspective the home screen can be split into 4 logical components: 
+
+1. **Home screen**: a hybrid app that is in charge of fetching the home screen configuration for the tenant and dynamically displaying the widgets.
+2. **Widgets**: Vue.js components that are loaded dynamically by the home screen and rendered. If you are not familiar with Vue.js, we recommend reading up on it [here](https://vuejs.org/v2/guide/).
+3. **Configurations**: Tenant admins can not only configure which widgets are shown on the home screen but also specify input parameters to the widgets themselves. Configurations are also loaded by the home screen on start up.  
+4. **HomeScreenSDK**: an SDK that glues everything together by allowing widgets to: 
+    * Register themselves onto the home screen
+    * Access the [Beekeeper Javascript SDK](https://developers.beekeeper.io/v2/welcome/javascript-sdk)
+    * Access the mobile bridge (TODO: add link to separate page)
```

Comment:
This is something you're planning to do at a later point?

Reply:
this I will change after discussing with Florian